scripts/prepare-release.js

```diff
@@ -1,9 +1,14 @@
-const { exec } = require('./utils');
+const { exec } = require("./utils");
```

Comment:
Why are you using "" instead of '' ?

Reply:
As @pmareke is saying some formatting tool changed, not only here, but every single quote (') to double quotes ("). Was this inteded?

Reply:
Apparently I formatted this file using prettier defaults. I have reformatted it with our code style. It might be interesting to add a global `.prettierrc` file in the root folder.